extensions/debug/deployment/src/main/java/org/apache/camel/quarkus/component/debug/deployment/DebugProcessor.java

```diff
@@ -16,18 +16,29 @@
  */
 package org.apache.camel.quarkus.component.debug.deployment;
 
+import java.util.HashSet;
+import java.util.LinkedHashSet;
 import java.util.function.BooleanSupplier;
 
+import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.builditem.AllowJNDIBuildItem;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.SystemPropertyBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
 import io.quarkus.runtime.LaunchMode;
 import org.apache.camel.quarkus.component.debug.DebugConfig;
 import org.apache.camel.quarkus.core.deployment.spi.CamelServiceDestination;
 import org.apache.camel.quarkus.core.deployment.spi.CamelServicePatternBuildItem;
 import org.eclipse.microprofile.config.ConfigProvider;
 
 class DebugProcessor {
+
+    private static final String[] SERIALIZATION_CLASSES = {
```

Comment:
Since the management extension already produces `CamelSerializationBuildItem`. We could maybe add these into the common set of classes we configure for serialization, WDYT?

https://github.com/apache/camel-quarkus/blob/main/extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelSerializationProcessor.java#L40

Reply:
👍 it makes sense, thx

Reply:
Should be good now